### DISCUSSION TITLE
error running for imputed VCF

### DISCUSSION THREAD

**Discussion by hereagain-Y** [PGScatalog/pgsc_calc#270] [General]
Hi, when I was trying to my vcf file, I first used plink2 to generate pfiles, when I used command: nextflow run pgscatalog/pgsc_calc -profile singularity --input /home/dyao8/test/chrm20_samplesheet.csv --target_build GRCh37  --pgs_id PGS00001
8 -latest -r v2.0.0-alpha.4, it keeps showing error at the PGSCATALOG_PGSCALC:PGSCALC:MATCH:MATCH_COMBINE (testchr20)  stage with   MATCH_COMBINE:
      pgscatalog_utils: $(echo $(python -c 'import pgscatalog_utils; print(pgscatalog_utils.__version__)'))
  END_VERSIONS

Command exit status:
  1; Any ideas on why there is error?

**Comment by smlmbrt** [comment]
Could you update to the latest version of the pipeline `v2.0.0-alpha.5` and see if that fixes? It's also possible that it has that exit status because there weren't enough variant matches, perhaps lowering min_overlap could help.
